### DISCUSSION TITLE
Show Achievements on my profile

### DISCUSSION THREAD

**Discussion by ElyasShamal** [community/community#56462] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I hide one of my Achievements using the eye icon  now I am not able to add it back to my profile anyone how to fix this issue

**Comment by radar-blip** [comment, marked as answer]
Try this:
https://github.com/ElyasShamal?tab=achievements

**Reply by ElyasShamal** [reply]
Thank you

**Reply by radar-blip** [reply]
No problem. Glad to help! 👍🏻
